caproto/threading/pyepics_compat.py

```diff
@@ -962,29 +962,27 @@ def disconnect(self):
 
 
 def get_pv(pvname, *args, context=None, connect=False, timeout=5, **kwargs):
-    if context is None:
```

Comment:
Thank you for fixing this.  It was on my mental to-do list but never made it up to being done!